### ISSUE TITLE
Base and recommends packages have invalid links to CRAN

### ISSUE THREAD

**Issue by eliocamp** [deepayan/r-refmans#2]
The base and recommends packages have links to CRAN (for example, https://cran.r-project.org/package=compiler) that are 404. I don't know if there's a canonical landing page for these packages. If there is not, they could either be changed to the R package main page or removed altogether.

**Comment by deepayan** [comment]
Yes, these will need to be handled somehow. Currently, links to base packages also have no place to go to on CRAN (i.e., there are no PDF manuals for base packages).
